docs/network/smart-rollup-nodes.md

```diff
@@ -0,0 +1,86 @@
+---
+title: Running an Etherlink Smart Rollup node
+---
+
+Etherlink uses Smart Rollup nodes as a bridge between the EVM transactions on the EVM nodes and Tezos layer 1.
+Public Smart Rollup nodes for Etherlink are not yet available, so you must run your own if you want to participate in the Etherlink network.
+
+The Smart Rollup node runs the _kernel_, which is a Rust program compiled in WASM implementing the semantics of Etherlink blocks and transactions.
+
+To start, the node uses the _installer kernel_, which is a compressed version of the kernel that provides only enough information to install the original kernel.
+The data for the original kernel is stored in separate files called _preimages_.
+For more information about installer kernels and preimages, see the tutorial [Deploy a Smart Rollup](https://docs.tezos.com/tutorials/smart-rollup) on docs.tezos.com.
+
+## Prerequisites
+
+- Make sure you understand the interaction between different nodes as described in [Etherlink architecture](./architecture).
+- Get access to a Tezos layer 1 archive node.
+You can use a public layer 1 node or set up your own as described in [Installing Octez](https://tezos.gitlab.io/introduction/howtoget.html) in the Octez documentation.
+
+## Running the Smart Rollup node
+
+You can start the Smart Rollup node with a snapshot of the Etherlink state to prevent it from having to compute the state from Etherlink genesis.
+
+For simplicity, these steps show how to run the smart rollup node in observer mode:
+
+1. Get a built version of the Smart Rollup node binary, named `octez-smart-rollup-node`.
+The best place to get the most recent binary files to use with Etherlink is https://gitlab.com/nomadic-labs/tezos/-/releases.
```

Comment:
I was linking to that page because releases from tezos/tezos page do not have the EVM node yet, but  this page is just about the Smart Rollup node, so I'll set it back to tezos/tezos.